src/scss/utilities/variables/_colors.scss

```diff
@@ -11,6 +11,6 @@ $soe-color-orange: #ffbd54;
 // colors that will be added to Decanter later and converted to using those values instead later:
 $su-color-bright-palo-alto: #017e7c;
 $su-color-bright-purple: #620059;
-$su-color-bay: #016895; // blue and bright blue are already in use in decanter colors
+$su-color-bay: #016895; // perhaps 'bay blue'? blue and bright blue are already in use in decanter colors
```

Comment:
I like bay blue or ocean blue. For me, bay doesn't mean blue on its own. :)

Reply:
updated as we discussed!